src/pages/styles.module.css

```
@@ -32,13 +40,33 @@
 }
 
 .featureImage {
-  height: 130px;
-  width: 130px;
+  height: 90px;
+  width: 90px;
 }
 
 .featuresGrid {
   display: grid;
   grid-template-columns: repeat(auto-fit, minmax(360px, 1fr));
-  grid-auto-rows: 330px;
-  gap: 4px;
+  grid-auto-rows: 250px;
+  gap: 2.5rem;
+}
+
+/* new styles */
+
+.padmd {
+  padding: 0 2rem;
+}
+
+.featuresp {
```

```suggestion
.featuresDescription {
```

1. See my previous comments on semantic CSS vs utility -- this class is named after an HTML element (`p`), and I'd prefer to maintain the semantic-CSS approach already in place for this page, to avoid mixing metaphors.
2. We should also use a case that communicates word boundaries. The file is already using `camelCase` so I chose that for this suggestion; I don't actually care if we did `kebab-case` instead (e.g. `featuresDescription`.)

This would require corresponding changes in the `pages/index.js` file too.